lib/models/analytic/aiuta_analytic_event_type.dart

```diff
@@ -0,0 +1,39 @@
+enum AiutaAnalyticEventType {
+  // Pages
+  // Onboarding
+  WelcomePageEntered,
+  HowItWorksPageEntered,
+  BestResultPageEntered,
+  ConsentPageEntered,
+  // Picker
+  ImagePickerPageEntered,
+  LoadingTryOnPageEntered,
+  // Result
+  ResultPageEntered,
+  // History
+  HistoryPageEntered,
+
+  // Actions
+  // Onboarding
+  WelcomeScreenStartButtonClicked,
+  ConsentGiven,
+  // Picker
+  TryOnStarted,
+  NewPhotoStarted,
```

Comment:
We have `NewPhotoTaken`/`GalleryPhotosSelected` events - can reuse them or create new